notebooks/114-simplified-mode/114-simplified-mode.ipynb

```diff
@@ -0,0 +1,367 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# INT8 Quantization by POT in Simplified Mode tutorial\n",
+    "\n",
+    "This tutorial shows how to quantize a ResNet20 image classification model, trained on CIFAR10 dataset, using the Simplified Mode of OpenVINO's Post-Training Optimization Tool. \n",
+    "\n",
+    "With the Simplified Mode models can be quantized without writing an accuracy checker configuration file. The user just need to specify the folder with images, which will later be used as calibration dataset for POT.\n",
```

Comment:
I don't think it is a good idea to mention accuracy checker without even providing links to it. I would remove this mention at all and revise the information about Simplified mode. You can take it from here: https://github.com/openvinotoolkit/openvino/blob/master/tools/pot/docs/SimplifiedMode.md